smithy-aws-endpoints/src/main/resources/META-INF/smithy/aws.endpoints.smithy

```diff
@@ -0,0 +1,128 @@
+$version: "2.0"
+
+namespace aws.endpoints
+
+/// Marks a trait as an endpoints modifier defining trait.
+///
+/// The targeted trait must only be applied to service shapes,
+/// must be a structure, and must have the `trait` trait.
+@trait(
+    selector: "structure[trait|trait]",
+    breakingChanges: [{change: "presence"}]
+)
```

Comment:
It looks like instead of just making this one trait stable, all the other `@unstable` traits were removed as well. `@endpointsModifier` should be the only trait not marked with `@unstable`.